.travis.yml

```diff
@@ -8,13 +10,15 @@ notifications:
 node_js:
   - '6'
 before_install:
-  - npm i -g yarn
-  - npm i -g codecov
+  - npm i -g yarn codecov
 before_script:
   - npm prune
+  - sleep 15
```

Comment:
Copied from [here](https://docs.travis-ci.com/user/database-setup/#MongoDB-does-not-immediately-accept-connections).  May not be needed since I reorganized tests.  I'll try removing it.